### DISCUSSION TITLE
I am using primeNg Library in angular library and want to use angular library in multiple angular project

### DISCUSSION THREAD

**Discussion by shubham-akeo** [primefaces/community#987] [PrimeNG]
Hi all, 
As the title mentioned i am using primeNg library in angular library and that library i am using into multiple angular application, while i try to load primeNg component or module in angular application, i am getting error below mentioned at the compile time - 

**Error: src/app/app.component.ts:9:14 - error NG3004: Unable to import component Toast.
The symbol is not exported from G:/web/message/node_modules/primeng/toast/index.d.ts (module 'primeng/toast').**

Can anyone help me here. 
Thanks in advance

**Comment by cetincakiroglu** [comment]
If you want to use multiple libraries, here's what you need to do:
	1.	Download your libraries using npm, yarn, or your chosen package manager.
	2.	Save the styles in the angular.json file as specified in the documentation.
	3.	Import and use the modules you need.
Please note that when using multiple libraries, their styles may override each other and cause unintended results. It's unclear what the source of the error you mentioned above is. To better assist you, could you please share a StackBlitz example?

**Comment by shubham-akeo** [comment]
@cetincakiroglu  I have already added all the modules in module file to export and angular library angular.json file does not support style attribute. 

![image](https://github.com/primefaces/community/assets/75292869/bea5ae49-7cd1-4d68-a792-fca3702beba2)

**Reply by cetincakiroglu** [reply]
angular.json file does not support `style` it should be `styles`. Please follow the guide in https://primeng.org/installation

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity
